kuma/wiki/admin.py

```diff
@@ -260,12 +260,26 @@ def rendering_info(obj):
 
 
 def akismet_data_as_dl(akismet_data):
+    """Format Akismet data as a definition list."""
+    favorites = (
```

Comment:
I do not know what you mean by ordering in a dictionary rather than a list. Sample code?

I disagree that moving it to another file makes it easier to configure. It only make it harder to understand the code, and where to make changes.